python/tskit/trees.py

```diff
@@ -1022,14 +1022,19 @@ def draw(
             joining each node in the map to its parent. As for ``node_colours``,
             unspecified edges take the default colour, and ``None`` values result in the
             edge being omitted. (Only supported in the SVG format.)
+        :param map max_timescale: If None (the default), scale the tree so that it fits
```

Comment:
the type is float not map. I think we can call this ``time_scale``.

The scaling used to map tree sequence time values into image y-coordinates. If None or unspecified, this is set to the time of the oldest root in the tree.